src/requests/user.js

```diff
@@ -94,8 +94,7 @@ const list = function (
 	searchText = '',
 	organization_code = null,
 	tenant_code = null,
-	body = {},
-	userToken = ''
```

Comment:
@adithyadinesh0412 for sunbitd user fetch userToken is required, please dont remove if userToken is already added